lib/tasks/top_records.rake

```
@@ -0,0 +1,166 @@
+
+    # Write CSV
+    require "csv"
+    dir = File.dirname(output_path)
+    FileUtils.mkdir_p(dir) unless File.directory?(dir)
+
+    CSV.open(output_path, "w") do |csv|
+      csv << [
+        "Rank", "Record ID", "Artist", "Label", "Genre", "Format",
+        "Detail", "Condition", "Price Low", "Price High",
+        "Personal Value", "Adjusted Value",
+        "Discogs Lowest Price"
```

RuboCop (1.84.2)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)